gnosis/safe-react
# Align Safe Apps disclaimer

## What it solves
Resolves #2865 

## How this PR fixes it
Restore same margin values that this component had before removing the footer

## How to test it
Just open any of the environments and check the disclaimer at the bottom of the Safe Apps list

## Screenshots
Will add a screenshot once build ends
